miio/fan_common.py

```diff
@@ -0,0 +1,23 @@
+import enum
```

Comment:
Good catch with common functionalities :+1: The long-term plan is to have base classes for common device types, for both enforcing the API as well as making it easier for downstreams to integrate support for new devices.